File: 1-Iniciante/ex1067.c
#include <stdio.h>

main(){
    /*variaveis*/
    int final, i;

    /*entrada*/
    scanf("%d", &final);

    /*processamento & saída*/
    for(i = 1; i <= final; i++){
        if(i % 2 == 1){
            printf("%d\n", i);
        }
    }
}

File: 1-Iniciante/ex1067.js
// entrada
let input = require('fs').readFileSync('/dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd

// processamento & saida
let x = parseInt(input);
if(x % 2 === 1) x++;
for(let i = 1; i < x; i += 2) console.log(i);